src/main/java/de/doubleslash/keeptime/view/ReportController.java

```diff
@@ -282,7 +280,7 @@ public void updateItem(final LocalDate item, final boolean empty) {
    }
 
    private Button createDeleteWorkButton(final Work w) {
-      final Button deleteButton = new Button("", new FontAwesomeIconView(FontAwesomeIcon.TRASH));
```

Comment:
I think you are missing the icons for this controller? (Trash, Pencil, Clipboard)